src/content/docs/en/guides/integrations-guide/sitemap.mdx

```diff
@@ -475,6 +475,40 @@ export default defineConfig({
 
 The given configuration will generate sitemap files at `https://example.com/astronomy-sitemap-0.xml` and `https://example.com/astronomy-sitemap-index.xml`.
 
+### `namespaces`
+
+<p>
+
+**Type:** `{ news?: boolean; xhtml?: boolean; image?: boolean; video?: boolean; }` <br />
+**Default:** `{ news: true, xhtml: true, image: true, video: true }` <br />
+<Since v="3.6.0" pkg="@astrojs/sitemap" />
+</p>
+
+An object of XML namespaces to exclude from the generated sitemap.
+
+Excluding unused namespaces can help create cleaner, more focused sitemaps that are faster for search engines to parse and use less bandwidth. For example, if your site doesn't have news content, videos, or multiple languages, you can exclude those namespaces to reduce XML bloat.
```

Comment:
```suggestion
Excluding unused namespaces can help create more focused sitemaps that are faster for search engines to parse and use less bandwidth. For example, if your site doesn't have news content, videos, or multiple languages, you can exclude those namespaces to reduce XML bloat.
```
I love this paragraph, thank you so much for adding it! I think we can stick to just "more focused" to keep this... more focused! 😂